utm.yaml

```diff
@@ -3457,6 +3491,16 @@ paths:
         $ref: '#/components/schemas/EntityID'
       in: path
       required: true
+    - name: version
+      description: >-
+        This optional field not part of the original F3548 standard API allows a
+        remote USS to request a specific version of the operational intent
+        details.  USSs are not required to implement support for this field.
```

Comment:
Given there would be significant burden on USSs to maintain every past version of an operational intent, is it worth adding here that a USS is expected to only keep two versions - the current and future? 

Also, do we want to add that the anticipated effect of querying the future version is that it is a signal to the USS that that version has been accepted by the DSS?

Reply:
Makes sense. I've updated the description accordingly.